docs/bnb-greenfield/for-developers/blob-hub.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Blob Hub
+description: Build “Blob” Data Archive on Greenfield
+keywords: [ BNB Greenfield, NodeReal, Blob hub ]
+---
+
+# Blob Hub
+
+# What Is Data Archive Layer?
```

Comment:
I think we should split this page into two page: "Data Archive Layer" and `Blobhub`, as blobhub is an example of `Build on Data Archive Layer`

Reply:
splited